logic.js

```diff
@@ -0,0 +1,89 @@
+const myBooks = [
+  {
+    title: 'The Design of EveryDay Things',
+    author: 'Don Norman',
+  }, {
+    title: 'The Most Human Human',
+    author: 'Brian Christian',
+  }, {
+    title: '1984',
+    author: 'George Orwell',
+  }];
+
+const form = document.getElementById('form');
+const title = document.getElementById('title');
```

Comment:
- [ ] An error is traced in your _Logic.j_s file. Kindly endeavor to fix this issue.

**Error:**
![image](https://user-images.githubusercontent.com/12745474/148893624-6ca1d7ed-005b-4f7b-b281-af9c7475116d.png)

Reply:
About this suggestion, I can´t find the requirement asking for no error on JS console. That said, I believe this error will be solved by if the localStore bug. If that is not the case, I would kindly ask for a second opinion about this.

Reply:
Hi @sergiogval 
You are right! This request is supposed to be optional.
I apologize for omitting this important step during my review. Kindly note that this was not a deliberate attempt!

Cheers and happy coding! 😃